deploy/crds/ocmagent.managed.openshift.io_ocmagents_crd.yaml

```
@@ -29,10 +29,59 @@ spec:
           metadata:
             type: object
           spec:
-            description: OCMAgentSpec defines the desired state of OCMAgent
+            description: OcmAgentSpec defines the desired state of OcmAgent
+            properties:
+              environment:
+                description: Environment defines the cluster env where the ocm agent
```

Extremely minor nit, but could we capitalize OCM in all of its usages in descriptions?
ie.
```suggestion
                description: Environment defines the cluster env where the OCM agent
```